assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -373,14 +373,16 @@ export default createWidget("discourse-reactions-actions", {
 
         tempReactions.push(newReaction);
 
+        //sorts reactions and get index of new reaction
         const newReactionIndex = tempReactions
-          .sort((reaction1, reaction2) =>
-            reaction2.count - reaction1.count || reaction1.id > reaction2.id
-              ? 1
-              : reaction2.id > reaction1.id
-              ? -1
-              : 0
-          )
+          .sort((reaction1, reaction2) => {
+            if (reaction1.count > reaction2.count) return -1;
```

Comment:
@jjaffeux thoughts on the readability of this code?

Reply:
yes much better